src/render-delaying-services.js

```diff
@@ -19,16 +19,16 @@ import {getServicePromise} from './service';
 import {timerFor} from './timer';
 
 /**
- * A map of extensions that, if they're included on the page, must be loaded
- * before the page should be shown to users. The key is the extension name,
- * the value is the blocking service.
- * Do not add an extension unless absolutely necessary.
+ * A map of services that delay rendering. The key is the name of the service
+ * and the value is a DOM query which is used to check if the service is needed
+ * in the current document.
+ * Do not add a service unless absolutely necessary.
  * @const {!Object<string, string>}
  */
-const EXTENSIONS = {
-  'amp-accordion': 'amp-accordion',
-  'amp-dynamic-css-classes': 'amp-dynamic-css-classes',
-  'amp-experiment': 'variant',
+const SERVICES = {
+  'amp-accordion': '[custom-element=amp-accordion]',
+  'amp-dynamic-css-classes': '[custom-element=amp-dynamic-css-classes]',
+  'variant': 'amp-experiment',
```

Comment:
Unfortunately, this won't always work as expected. There's no guarantee the `body` element has been fully parsed yet, so it may not find an `<amp-experiment>` close to the bottom of the page. We've gotten away with the current because all the `<script>` tags are in the `<head>`, which is very likely parsed (though not guaranteed).